### DISCUSSION TITLE
Request Failed with status code 500

### DISCUSSION THREAD

**Discussion by heimdal305** [Security-Onion-Solutions/securityonion#2619] [Unsupported Versions]
Hi all, 

I installed SecOnion in standalone mode and I think I am kind of in a bind. Whenever I log in and go to alerts, I receive a Request failed with status code 500 error. I tried the deployment multiple times with same results. All components seems to have started up with no issues. I did get an error during install so I am not sure if this is what is causing the issue. 

Thank for your help.

<img width="1326" alt="Screen Shot 2021-01-14 at 3 15 21 AM" src="https://user-images.githubusercontent.com/77433385/104562776-c9cf3000-5616-11eb-8b76-b0e7bd658923.png">

<img width="1429" alt="Screen Shot 2021-01-14 at 3 14 00 AM" src="https://user-images.githubusercontent.com/77433385/104562623-99879180-5616-11eb-81ba-d87392439fe6.png">

Additional information:
https://x.x.x.x/api/events/?query=event.dataset:alert+AND++NOT+event.acknowledged:true+AND++NOT+event.escalated:true+AND+*+%7C+groupby+rule.name+event.module+event.severity_label&range=2021%2F01%2F13+02:54:07+AM+-+2021%2F01%2F14+02:54:07+AM&format=2006%2F01%2F02+3:04:05+PM&zone=America%2FNew_York&metricLimit=500&eventLimit=500
[Error] Failed to load resource: the server responded with a status of 500 () (events, line 0)

**Comment by weslambert** [comment]
Do you get results when you look in Kibana?  Are there any clues from the Elasticsearch logs?

**Reply by deannick** [reply]
I have this same error.  I reinstalled the system and it came back.  If I go to Kibana, I get {"message":"An internal server error occurred","statusCode":500,"error":"Internal Server Error"} or a blank page with red triangles with exclamation marks.

**Comment by innovate-support** [comment]
what's the status of your system?
`sudo so-status`

**Reply by deannick** [reply]
Everything is green and says ok.

**Reply by deannick** [reply]
I rebooted again today and now both elasticalertand logstash are in error. Last week I reinstalled, due to not knowing how to resolve this.

**Comment by heimdal305** [comment]
Mine is working fine now. I had to make sure that traffic is as being sent
to the monitoring interface correctly. That’s was the only change I made
then it started working as expected. It is kind of strange but...

On Mon, Feb 22, 2021 at 3:57 PM deannick <notifications@github.com> wrote:

> Everything is green and says ok.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/2619#discussioncomment-395112>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASOYUKNW2SXDZWFKTPG3XV3TALAK5ANCNFSM4WCBRQCA>
> .
>
-- 

*Earl Matthew*


0100010101100001011100100110110000

1000000000110100001010

**Reply by deannick** [reply]
Mine will work for a about a week then this will happen.  I have verified that my data is going to the correct interface.

**Reply by Git-Me-Some-Hub** [reply]
I seem to have the same issue. Mine will be working digesting, but then maybe when too much data is in there ElasticSearch/Alert fails. Moving it to some faster hard drives does not fix the issue. Will have to get digging in the logs.

**Reply by weslambert** [reply]
I'd check the Elasticsearch logs for clues.

**Reply by Git-Me-Some-Hub** [reply]
I would like to say I knew enough to find anything useful in there. It seems to be about a week of Span and Winbeat data coming in crashes my default standalone instance. Kind of thinking on moving on to another product. Just do not have the time, before 2.3 I had this up and running for months and months. I just ran soup in hopes that could help something, but after a reboot 19 hours later with all services up Kibana still says it is not ready... 

I have attached the log entries that I had continue over and over on 3-13 when my server just stopped ingesting data. I can create a new post instead of blowing up this one too. 
[Elastic_Error00.txt](https://github.com/Security-Onion-Solutions/securityonion/files/6160994/Elastic_Error00.txt)

**Reply by Git-Me-Some-Hub** [reply]
I saw nothing in the Kibana log file, so I restarted and now I see something, but not sure what migration it is talking about:
 
{"type":"log","@timestamp":"2021-03-18T03:00:49Z","tags":["warning","savedobjects-service"],"pid":292,"message":"Another Kibana instance appears to be migrating the index. Waiting for that migration to complete. If no other Kibana instance is attempting migrations, you can get past this message by deleting index .kibana_2 and restarting Kibana."}

**Reply by dougburks** [reply]
@Git-Me-Some-Hub Please see https://docs.securityonion.net/en/2.3/kibana.html#diagnostic-logging. If you have further questions or problems, please start a new Discussion. Thanks!
